src/pages/workspaces/workspace/analysis/Analyses.ts

```diff
@@ -263,22 +280,6 @@ const activeFileTransferMessage = div({
   span({ style: { fontWeight: 'bold', marginLeft: '0.5ch' } }, ['This may take a few minutes.'])
 ])
 
-export type DisplayAnalysisFile = AnalysisFile & {
```

Comment:
this was just adding the application to the AnalysisFile, and the AnalysisFile type type already has a tool param that is the correct toolLabel, so I removed this type.